.travis.yml

```diff
@@ -3,6 +3,9 @@ python:
   - 2.7
   - 3.4
   - 3.5
```

Comment:
```suggestion
```

Let's go ahead and get rid of these since they're EOL'd https://devguide.python.org/devcycle/#end-of-life-branches